File: GLLara/GLLNoSelectionViewController.m
//
//  GLLNoSelectionViewController.m
//  GLLara
//
//  Created by Torsten Kammer on 28.02.13.
//  Copyright (c) 2013 Torsten Kammer. All rights reserved.
//

#import "GLLNoSelectionViewController.h"

@interface GLLNoSelectionViewController ()

@end

@implementation GLLNoSelectionViewController

- (id)init
{
    self = [super initWithNibName:@"GLLNoSelectionView" bundle:nil];
    if (self) {
        // Initialization code here.
    }
    
    return self;
}

- (IBAction)help:(id)sender;
{
	NSString *locBookName = [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleHelpBookName"];
	[[NSHelpManager sharedHelpManager] openHelpAnchor:@"models" inBook:locBookName];
}

@end

File: GLLara/GLLNoSelectionViewController.swift
//
//  GLLNoSelectionViewController.swift
//  GLLara
//
//  Created by Torsten Kammer on 23.07.22.
//  Copyright © 2022 Torsten Kammer. All rights reserved.
//

import Cocoa

@objc class GLLNoSelectionViewController: NSViewController {
    
    convenience init() {
        self.init(nibName: "GLLNoSelectionView", bundle: nil)
    }
    
    @IBAction override func showContextHelp(_ sender: Any?) {
        let locBookName = Bundle.main.object(forInfoDictionaryKey: "CFBundleHelpBookName") as! NSString?
        NSHelpManager.shared.openHelpAnchor("models", inBook: locBookName as NSHelpManager.BookName?)
    }
}
